getting-started-guide/observability.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Axiom for observability"
+description: "This page explains how Axiom helps you leverage timestamped event data for observability purposes."
+sidebarTitle: Observability
+---
+
+Axiom helps you leverage the power of timestamped event data. A common use case of event data is observability (o11y) in the field of software engineering. Observability means measuring the health and performance of apps and distributed systems. The goal of collecting this information is to monitor the status of apps and systems, improve their quality, and fix errors.
```

Comment:
A more precise definition of observability that is usually reference is one’s ability to explain what is happening on the inside of a system just from observing it on the outside, or the ability to understand the behavior of (software) systems based on their outputs (telemetry, i.e. event data).